clang/test/CodeGen/profile-filter.c

```diff
@@ -9,6 +9,9 @@
 // RUN: echo -e "[clang]\nfun:test1\n[llvm]\nfun:test2" > %t-section.list
 // RUN: %clang_cc1 -fprofile-instrument=llvm -fprofile-list=%t-section.list -emit-llvm %s -o - | FileCheck %s --check-prefix=SECTION
 
+// RUN: echo -e "[coldcov]\nfun:test*\n!fun:test2" > %t-cold-func.list
+// RUN: %clang_cc1 -fprofile-instrument=coldcov -fprofile-list=%t-cold-func.list -emit-llvm %s -o - | FileCheck %s --check-prefix=COLDCOV
```

Comment:
```suggestion
// RUN: %clang_cc1 -fprofile-instrument=coldcov -fprofile-list=%t-cold-func.list -emit-llvm %s -o - | FileCheck %s --check-prefix=COLDCOV --implicit-check-not=noprofile
```

Can we do this to avoid the `CHECK-NOT` lines?

Reply:
There is also a explicit check noprofile ("// COLDCOV: noprofile"), so I guess `implicit-check-not` doesn't work here.